tests/rest_scripts/calculate_crc64nvme.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Versity Software
+# This file is licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+source ./tests/rest_scripts/rest.sh
+
+if [ -z "$TEST_FILE_FOLDER" ] || [ -z "$DATA_FILE" ]; then
+  log_rest 2 "TEST_FILE_FOLDER and DATA_FILE params required for crc64nvme checksum (for python virtual env)"
+  exit 1
+fi
+if ! python3 -m venv "$TEST_FILE_FOLDER/env" 1>/dev/null; then
+  log_rest 2 "error creating python virtual environment"
+  exit 1
+fi
+if ! source "$TEST_FILE_FOLDER/env/bin/activate" 1>/dev/null; then
+  log_rest 2 "error activating virtual environment"
+  exit 1
+fi
+if ! python3 -m pip install awscrt 1>/dev/null 2>&1; then
+  log_rest 2 "error installing awscrt"
+  exit 1
+fi
+if ! checksum_decimal=$(python3 -c "
+import sys
+from awscrt import checksums
+
+with open(sys.argv[1], 'rb') as f:
```

Comment:
This seems like something that could be built into the test vm itself, so that we don't have to keep installing it.  Perhaps as a pre-built script that is installed and can then be executed as needed?